gateway-service-definitions/src/main/resources/services/airflow/1.10.4/service.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<service role="AIRFLOWUI" name="airflow-ui" version="1.10.14">
```

Comment:
This doesn't match the name of the folder structure. `services/airflow` vs `services/airflowui`.

Reply:
Also `1.10.14` doesn't match `1.10.4`. Is there a reason we need to go to the third version number? Any reason this wouldn't work with Airflow 1.10.0 or higher?

Reply:
It was remiss of me, I have modified the name of the folder. 
I will supplement the tests of airflow release version since 1.10.